docker-compose.yaml

```diff
@@ -1,24 +1,37 @@
-version: "3.3"
+########################### COMMON SETTINGS
+
+x-common-default: &common-default
+    security_opt:
+        - no-new-privileges:true
+    restart: unless-stopped
```

Comment:
I think it would be better to duplicate these settings for each service, for the following reasons:

- It makes the `docker-compose.yaml` file more readable, especially for beginners;
- It makes it easier to copy the services and integrate them into an existing Docker Compose file.

Reply:
I have removed the common-default mapping from the Docker Compose.